check.sh

```
@@ -636,161 +681,159 @@ show_installed_runtime_upgrade_hint() {
+runtime_skill_root="${TARGET_ROOT}/${startup_runtime_target_rel}"
 runtime_nested_skill_root="${runtime_skill_root}/bundled/skills/vibe"
+runtime_nested_skill_presence_policy='optional'
+runtime_nested_skill_required='false'
+if [[ -f "$repo_governance_path" ]]; then
+  topology_required="$(json_query_scalar_from_file "$repo_governance_path" 'mirror_topology.targets.1.required' 2>/dev/null || true)"
+  topology_presence_policy="$(json_query_scalar_from_file "$repo_governance_path" 'mirror_topology.targets.1.presence_policy' 2>/dev/null || true)"
+  if [[ -n "$topology_presence_policy" ]]; then
+    runtime_nested_skill_presence_policy="$topology_presence_policy"
+  fi
+  if [[ "$topology_required" == 'true' || "$runtime_nested_skill_presence_policy" == 'required' ]]; then
+    runtime_nested_skill_required='true'
```

Shellcheck (0.11.0)

[warning] runtime_nested_skill_required appears unused. Verify use (or export if used externally).

(SC2034)